tests/test_process.py

```diff
@@ -2,7 +2,10 @@
 
 
 def test_process_is_pid_1(beat):
-    assert beat.process.pid == 1
+    if beat.name == 'auditbeat':
+        assert beat.process.pid >= 1
```

Comment:
Is there ever a case where a process could have a PID that is not `>= 1`?

Did you mean `> 1`?

Reply:
excellent point. fixed in 73b936e